deploy/connect.yaml

```diff
@@ -291,6 +291,11 @@ objects:
         transforms.transformRunEvent.table: runs
         transforms.transformRunEvent.topic: ${KAFKA_TOPIC_PREFIX}platform.playbook-dispatcher.runs
 
+        hosttransforms: transformRunHostsEvent
```

Comment:
This is not correct. The connector configuration syntax is puzzling. It seems correct in `examples/connector-local.json`

https://docs.confluent.io/platform/current/installation/configuration/connect/source-connect-configs.html#sourceconnectorconfigs_transforms

Reply:
I think I see how this needs to be configured.  Looks like the value of the "transforms" needs to be used as part of  the property name to define the type, topic, and table.

Reply:
@Jharting would you be able to take a quick look at the config?  I think I have it right now.